src/main/Steering.cpp

```diff
@@ -0,0 +1,42 @@
+#include "Steering.h"
+#include "Arduino.h"
+#include <Servo.h>
+#include <ESP32Servo.h>
+
+
+//Define servo motor classes
+Servo leftMotor;
+Servo rightMotor;
+//Change Pulse Width frequency as needed
+#define minUs 1000; 
+#define maxUs 2000;
+//PWM object
+ESP32PWM pwm;
+
+void motorSetup(){
+  leftMotor.attach(11,minUs,maxUs);
+  rightMotor.attach(12,minUs,maxUs);
+  pwm.attachPin(11,1000); //1Khz - Is this correct?
+  pwm.attachPin(12,1000); //1Khz - Is this correct?
+  leftMotor.setPeriodHertz(50);      // Standard 50hz servo - Is this correct?
+	rightMotor.setPeriodHertz(50);      // Standard 50hz servo - Is this correct?
+}
```

Comment:
i think it would be better if we give some _names_ instead of keeping them as _magic_ numbers. 
```#define ... ``` perhaps?